bin/plugin/commands/packages.js

```diff
@@ -297,59 +336,64 @@ async function prepareForPackageRelease( minimumVersionBump, isPrerelease ) {
 		abortMessage
 	);
 
-	await updatePackages(
+	const commitHash = await updatePackageChangelogs(
 		gitWorkingDirectoryPath,
 		minimumVersionBump,
-		isPrerelease,
 		abortMessage
 	);
 
-	// Push the local changes
-	abortMessage = `Aborting! Make sure to push changes applied to WordPress release branch "${ releaseBranch }" manually.`;
 	await runPushGitChangesStep(
 		gitWorkingDirectoryPath,
 		releaseBranch,
 		abortMessage
 	);
 
-	abortMessage = 'Aborting! The release is finished though.';
-	await runCleanLocalFoldersStep( temporaryFolders, abortMessage );
-}
+	abortMessage = `Aborting! The release branch is synced. Make sure to publish the npm packages manually and backport the changelog commit (${ formats.title(
+		commitHash
+	) }) to master.`;
 
-async function prepareLatestDistTag() {
-	log(
-		formats.title(
-			'\n💃 Time to publish WordPress packages to npm 🕺\n\n'
-		),
-		'Welcome! This tool is going to help you with preparing everything for publishing a new stable version of WordPress packages.\n',
-		"To perform a release you'll have to be a member of the WordPress Team on npm.\n"
+	await publishPackages(
+		gitWorkingDirectoryPath,
+		minimumVersionBump,
+		abortMessage
 	);
 
-	await prepareForPackageRelease( 'patch' );
+	abortMessage = `Aborting! The packages have published though. Make sure to backport the changelog commit (${ formats.title(
+		commitHash
+	) }) to master.`;
 
-	log(
-		'\n>> 🎉 WordPress packages are ready to publish!\n',
-		'You need to run "npm run publish:prod" to release them to npm.\n',
-		'Let also people know on WordPress Slack when everything is finished.\n'
+	await runPushGitChangesStep(
+		gitWorkingDirectoryPath,
+		releaseBranch,
+		abortMessage
+	);
+
+	console.log(
```

Comment:
There are two commits to backport here:
- the one with updates to changelog files
- the one with package version changes applied by Lerna

Reply:
Yes, I know that last is not straightforward to find. Need to use git log maybe.